coalib/results/result_actions/OpenEditorAction.py

```diff
@@ -43,6 +43,10 @@
         'file_arg_template': '+{line},{column} {filename} ',
         'gui': False
     },
+    'emacs': {
+        'file_arg_template': '+{line}:{column} {filename}',
+        'gui': False
+    },
```

Comment:
@pratyushprakash Yeah, add `emacsclient` because some people don't want to have multiple instances of `emacs` servers